src/libraries/System.Private.CoreLib/src/System/ReadOnlySpan.cs

```diff
@@ -147,7 +147,7 @@ public int Length
         }
 
         /// <summary>
-        /// Gets a value indicating whether this read-only span is empty.
+        /// Gets a value indicating whether this <see cref="ReadOnlySpan"/> is empty.
```

Comment:
@AaronRobinsonMSFT Somehow, I this isn't recognized. Maybe the Full-Qualified would be better.

Reply:
I think this would need to be `ReadOnlySpan{T}`.

Reply:
@xtqqczze Right 🤦🏼‍♂️ 

```suggestion
        /// Gets a value indicating whether this <see cref="ReadOnlySpan{T}"/> is empty.
```

Reply:
Actually, the code reference seems redundant because `IsEmpty` is already a member of `ReadOnlySpan<T>`.

Reply:
https://github.com/dotnet/runtime/blob/0a9a592bb2bac9af67195b9db5b3101462105f9f/src/libraries/System.Private.CoreLib/src/System/ReadOnlySpan.cs#L140-L143

Reply:
That's true, but I think I've seen a lot of similar code.

I'd say we'll let @AaronRobinsonMSFT decide.

Reply:
@xtqqczze Your point about redundancy is fair. @deeprobin is also correct that we do have this pattern in various places so it isn't without precedent. In order to reconcile the two ideas it is helpful to know there is an effort to make all triple-slash references the true source of truth for documentation instead of XML files that regularly need to be refreshed in https://github.com/dotnet/dotnet-api-docs.

This is a long-term goal and I'm not sure when it will be achieved, but a goal none the less. In the spirit of that effort, ensuring the documentation is using support triple-slash syntax so that documentation is not only helpful but also links properly when generated is encouraged.